lxd/devices.go

```diff
@@ -559,7 +559,7 @@ func deviceEventListener(s *state.State) {
 
 			logger.Debugf("Scheduler: network: %s has been added: updating network priorities", e[0])
 			deviceNetworkPriority(s, e[0])
-			networkAutoAttach(s.DB.Cluster, e[0])
+			_ = networkAutoAttach(s.DB.Cluster, e[0])
```

Comment:
This error should be logged as a warning as the event hasn't been handled successfully.